Riot/Modules/TabBar/TabBarCoordinator.swift

```diff
@@ -702,6 +709,11 @@ extension TabBarCoordinator: RoomCoordinatorDelegate {
     func roomCoordinatorDidLeaveRoom(_ coordinator: RoomCoordinatorProtocol) {
         // For the moment when a room is left, reset the split detail with placeholder
         self.resetSplitViewDetails()
+        if BuildSettings.appActivityIndicators {
```

Comment:
Can we add a verb to these build settings to follow the local conventions e.g. `useAppActivityIndicators` or something?